docs/README.md

```diff
@@ -0,0 +1,46 @@
+## Access documentation locally
```

Comment:
There are two audiences here:
- 1) people looking to contribute/develop/write docs
- 2) people trying to read pygame docs.

Doesn't have to be in this PR, but we probably also need something for people looking into the docs folder looking for pygame docs.